nav2_bringup/README.md

```diff
@@ -2,121 +2,86 @@
 
-Notes: (December 2018, Crystal Release)
-* We recommend doing this on a Ubuntu 18.04 installation. We’re currently having build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
+Notes: (June 2019, Dashing Release)
+* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
 * This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
 * It is recommended to start with simulation using Gazebo before proceeding to run on a physical robot
 
 Install and build our code by following this guide:
 https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md
 
-## Launch Navigation2 in simulation with Gazebo (first time users)
+## Launch Navigation2 in simulation with Gazebo
 ### Pre-requisites:
 * Gazebo installed on the system
-* gazebo_ros_pkgs for ROS2 installed on the system
+* gazebo\_ros\_pkgs for ROS2 installed on the system
 * A Gazebo world for simulating the robot (see Gazebo tutorials)
 * A map of that world saved to a map.pgm and map.yaml (see ROS Navigation tutorials)
 
-### Terminal 1: Launch Gazebo and Rviz2
+### Terminal 1: Launch Gazebo
 
 Example: See [turtlebot3_gazebo models](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo/models) for details
+
 ```
 export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:<full/path/to/my_robot/models>
-ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>
+gazebo --verbose -s libgazebo_ros_init.so <full/path/to/my_gazebo.world>
 ```
 
 ### Terminal 2: Launch your robot specific transforms
 
 Example: See [turtlebot3_gazebo](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo) for details
 
-`ros2 launch turtlebot3_bringup burger_state_publisher.launch.py`
-
-### Terminal 3: Launch map_server and AMCL
-
 ```
-# Set the tf publisher node to use simulation time or AMCL won't get the transforms correctly
-ros2 param set /robot_state_publisher use_sim_time True
-# Launch map_server and AMCL, set map_type as "occupancy" by default.
-ros2 launch nav2_bringup nav2_bringup_1st_launch.py map:=<full/path/to/map.yaml> map_type:=occupancy use_sim_time:=True
+source turtlebot3_ws/install/setup.bash
+export TURTLEBOT3_MODEL=waffle
+ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time:=True
 ```
-In RVIZ:
-* Make sure all transforms from odom are present. (odom->base_link->base_scan)
-* Localize the robot using “2D Pose Estimate” button.
 
-### Terminal 4:
-Run the rest of the Navigation2 bringup
-
-`ros2 launch nav2_bringup nav2_bringup_2nd_launch.py use_sim_time:=True`
-
-### Terminal 5:
-Set the World Model and the two costmap nodes to use simulation time
+### Terminal 3: Launch navigation2
```

Comment:
You seemed to have left out the `nav2_bringup_1st_launch.py` launch file as part of the instructions. Also, I think we should make it clear what each launch files does, both with the documentation and perhaps a more descriptive launch name.

Reply:
In my comments at the top I tried to explain that this is included but not needed in the README. It needs to be validated with SLAM.

I agree on the name, I thought that could be a follow up also. I believe @SteveMacenski had a PR along those lines. Once this merges we could make that change. I was re-adding them with the same names as before for now.

Reply:
I had mistakenly put nav2_bringup_1st_launch.py instead of nav2_bringup_launch.py which is why this was confusing. I've fixed it in the latest push.